Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -4326,6 +4326,12 @@ - (void)resendGroupUpdateForErrorMessage:(TSErrorMessage *)message
            }];
 }
 
+- (void)conversationColorWasUpdated
+{
+    [self.conversationStyle updateProperties];
+    [self.headerView updateAvatar];
```

Comment:
Don't we need to reload our collection view?

Reply:
It's apparently not necessary, but I've added it anyway to be safe.